torch_geometric/nn/glob.py

```diff
@@ -0,0 +1,35 @@
+from torch_geometric.deprecation import deprecated
+from torch_geometric.nn.aggr import (
+    AttentionalAggregation,
+    GraphMultisetTransformer,
+    Set2Set,
+    SortAggr,
+)
+
+Set2Set = deprecated(
+    details="use 'nn.aggr.Set2Set' instead",
+    func_name='nn.pool.Set2Set',
```

Comment:
The deprecations are moved back to `nn.glob.*`. Should we change back from `nn.pool.*` to  `nn.glob.*`? Sorry about that.

Reply:
I think this is correct. We provide backward compatibility in `nn/glob.py` but deprecate its use.